docs/scraping-service.md

```diff
@@ -0,0 +1,125 @@
+# Scraping Service Mode
+
+Scraping Service Mode is a third operational mode of the Grafana Cloud Agent
+that allows for clustering a set of Agent processes and distributing scrape load
+between them.
+
+Determining what to scrape is done by storing instance configuration files to an
+[API](./api.md) which then stores the configuration files in a KV store backend.
+All agents in the cluster must use the same KV store so they read the same set
+of config files.
+
+The "instance" refers to the process of:
+
+- Service discovery for all `scrape_configs` within that loaded config
+- Scrapes metrics from all discovered targets
+- Stores data in its own Write-Ahead Log specific to the loaded config
+- Remote Writes scraped metrics to the configured `remote_write` clients
+  specified within the loaded config.
+
```

Comment:
Maybe add one final line after the bullet points `This means that, for each unique config file an agent is supposed to collect metrics for, there is an "instance" running each of these above processes`

Reply:
Oh okay, you clarify that in the next sentence. So maybe just consider reordering here, but I don't have a concrete suggestion for that.